src/App.js

```diff
@@ -1,21 +1,80 @@
 import BookList from './components/BookList'
+import Navigation from './components/Navigation'
 import styled from 'styled-components/macro'
+import { Route, Switch } from 'react-router-dom'
+import { useState } from 'react'
 
 function App({ data }) {
+  const [filteredBooks, setFilteredBooks] = useState(data)
+  const [readingStatus, setReadingStatus] = useState('')
+
+  function handleActiveReadingStatus(status, books) {
```

Comment:
I think the function name is not 100% good. It kind of does two things. First it changes the `readingStatus` and sets the `filteredBooks`. I can't really think of a good name myself though 😄.

Reply:
Yes you're right! Maybe `handleBookList` would be better?